reference-architecture/day2ops/scripts/project_export.sh

```diff
@@ -407,4 +407,7 @@ deployments
 replicasets
 poddisruptionbudget
 
+echo "Removing empty files"
+find ${PROJECT} -type f -empty -delete
```

Comment:
If project variable is provided by user ($1) it should be definitely quoted, as the path may contain white spaces. This is definitely important when deleting something. If I can recommend, use this tool to check your bash scripts - https://www.shellcheck.net/ It's even in Fedora repos ($ sudo dnf install ShellCheck ).